File: Data Structures & Algorithms/subsets/submission-0.cpp
class Solution 
{
public:
 void backtrack(  vector<vector<int>>&result, vector<int>&temp,vector<int>& nums,int index)
{
     result.push_back(temp);
    for(int i=index;i<nums.size();i++)
    {
        //inlcude it 
        temp.push_back(nums[i]);
        backtrack(result,temp,nums,i+1);
       
    //not include 
        // temp.remove(temp.size()-1);
    temp.pop_back();
    }

}
    vector<vector<int>> subsets(vector<int>& nums)
     {
        // vector<int>result;
        vector<vector<int>> result;
        vector<int>temp;
        backtrack(result,temp,nums,0);
        return result;
    }
};

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:

        n=len(nums)
        mp = {}

        for i in range(0,n):
            mp[nums[i]]=mp.get(nums[i],0)+1;


        for key,value in mp.items():
            if value>n/2:
                return key
        
        return -1;
        